src/views/Delegate.vue

```diff
@@ -101,7 +111,54 @@ function clearDelegate(id, delegate) {
   modalOpen.value = true;
 }
 
-const { profiles, addressArray } = useProfiles();
+async function getDelegatesWithScore() {
+  delegatesLoading.value = true;
+  try {
+    const { delegations } = await getDelegatesBySpace(
+      space.value.network,
+      space.value.id
+    );
+
+    const uniqueDelegators = Array.from(
```

Comment:
I would have improved readability by just using lodash. 

uniqBy(delegations, 'delegate')

Reply:
Yes, I will open an issue to replace `new Set` by lodash `uniqBy`  in our codebase